library/src/cache/patch_manager.rs

```diff
@@ -275,6 +275,19 @@ impl PatchManager {
             }
         }
     }
+
+    /// Deletes all patch artifacts with numbers less than patch_number.
+    fn delete_patch_artifacts_older_than(&mut self, patch_number: usize) -> Result<()> {
```

Comment:
We've talked about moving away from patch_number (which I think it just a name?) to some sort of patch_id?  Obviously that's not the state today.  Obviously we will still need some way to compare them by order.

Reply:
I'm not sure I recall – was this to allow iOS and Android to each have sequential patch numbers?

Reply:
To allow them to have the same patch number I believe.

Reply:
Yep, that's what I meant. I think the shift from "patch number as identifier" to some other ordered patch identifier will be pretty straightforward (maybe just `platformN` for patch number `N` on `platform`?) but definitely a bigger project than what I'm doing here.